### DISCUSSION TITLE
Google map和蛋白質計畫的大功臣 deepmind

### DISCUSSION THREAD

**Discussion by wu51000** [1101-GS3073-C/Python-Community-GS3073-C#739] [General]
上一篇題到的google map 使用的AI技能
也是deepmind協助運作
而這次deepmind則預測出2.14億種的蛋白質
以前在尚未ＡＩ化的時代裡只能人工測量
Deepmind的創新造福了醫學和藥學界的人們
患有蛋白質相關疾病的人也因為資料庫的增加
而更有機會治癒那些難纏的疾病
除此之外我個人覺得更了解蛋白質
也可以增加飲食上面食材的配置
了解攝取多少與哪種類的蛋白質對人體最有幫助
https://technews.tw/2022/08/03/deepmind-alphafold/

**Comment by yjpxi** [comment]
透過AI深入了解蛋白質在人體中的作用，我們可以更科學地規劃飲食，確保攝取到適合的蛋白質種類和數量，以促進健康。
